boefjes/boefjes/plugins/kat_webpage_analysis/analyze_page/normalizer.json

```diff
@@ -1,7 +1,7 @@
 {
-  "id": "kat_website_software_normalize",
+  "id": "kat_webpage_analysis_wappalyzer_normalize",
```

Comment:
I think it would be better if the directory of the boefje/normalizer matches with the id of the boefje/normalizer like we usually do.

I also think that normalizer would be better placed in the boefjes/plugins directory and not in boefjes/plugins/kat_webpage_analysis directory. It makes it easier to find, but there is also only a loose connection between boefje and normalizer. In the future there might be another boefje that also produces `openkat-http/response` and then it also wouldn't make a lot of sense that the normalizer inside the directory of another boefje. It also would make it easier to create a container image with the just the boefjes.

Reply:
The reason that it was placed under `kat_webpage_analysis` was because they share a common function, namely analyzing webpages. However, you're point about its high coupling is a good one, hence I moved the directory to the plugins base folder.